core/src/test/scala/org/apache/spark/io/ChunkedByteBufferSuite.scala

```diff
@@ -56,6 +56,12 @@ class ChunkedByteBufferSuite extends SparkFunSuite {
     assert(chunkedByteBuffer.getChunks().head.position() === 0)
   }
 
+  test("writeFully() can write buffer which is larger than bufferWriteChunkSize correctly") {
```

Comment:
nit: Would it be possible to add `SPARK-24107: ` into the start of the string? It would help us connect a UT with JIRA entry.

Reply:
Done. Thanks